README.md

```diff
@@ -66,112 +52,98 @@ The `-d` option will output a detailed JSON report of the internal C2PA structur
 
-When using a JSON file, the parent file can be specified by passing `-p` or `--parent` with the path to the file. This allows using the same manifest definition with different parent assets.
+```shell
+c2patool sample/test.json -p original.jpg -o edited_image.jpg
+```
 
-#### Usage notes
+#### Manifest preview feature
 
-If you are not changing an asset and just adding C2PA data, use an existing output file and no parent. _Note that this will replace any existing C2PA data in the existing output file._ For instance:
+If the output file is not specified, the tool will generate a preview of the generated manifest. This can be used to make sure you have formatted the manifest definition correctly.
 
 ```shell
-c2patool sample/test.json -o existing.jpg
+c2patool sample/test.json
 ```
+#### Shortcut feature
 
-If you have edited an asset and want to add C2PA data to it, pass the original as the parent and put the edited file at the output location to have the C2PA data added.
+If the output file does not exist, and a parent exists, the parent file will be copied to the output location and then updated.
 
 ```shell
-c2patool sample/test.json -p original.jpg -o image-with-c2pa.jpg
+c2patool sample/test.json -p original.jpg -o copy_of_original.jpg
 ```
 
-## Appendix
-
-### Creating and using an X.509 certificate
+#### No parent feature
 
-Before you can add a manifest, you need to create an X.509 certificate. You can specify the path to the cert files in the following configuration fields:
-
-- `private_key`
-- `sign_cert`
-
-If you are using a signing algorithm other than the default `ps256`, you will need to specify it in `alg`, which can be set to one of the following:
-
-- `ps256`
-- `ps384`
-- `ps512`
-- `es256`
-- `es384`
-- `es512`
-- `ed25519`
-
-The specified algorithm must be compatible with values of `private_key` and `sign_cert`.
-
-The key and cert can also be placed in the environment variables `C2PA_PRIVATE_KEY` and `C2PA_PUB_CERT`. These two variables are used to set the private key and public certificates. For example, to sign with es256 signatures using the content of a private key file and certificate file, you would run:
+If the output file exists and no parent is specified, the output file will be updated with a manifest created from the manifest definition only. Note that in this case, any previous manifest data in the output file will be replaced.
 
 ```shell
-set C2PA_PRIVATE_KEY=$(cat my_es256_private_key)
-set C2PA_PUB_CERT=$(cat my_es256_certs)
+c2patool sample/test.json -o new_manifest.jpg
 ```
 
-Both the `private_key` and `sign_cert` should be in PEM format. The `sign_cert` should contain a PEM certificate chain starting for the end-entity certificate used to sign the claim ending with the intermediate certificate before the root CA certificate. See the ["sample" folder](https://github.com/contentauth/c2patool/tree/main/sample) for example certificates.
+#### Example of a Manifest Definition file
```

Comment:
Given our decision to use sentence case, Manifest and Definition should not be capitalized.